### PR TITLE
Capstone 1.(Spytech Web Development Summit).

### PR DESCRIPTION
**Pull request by spydaspider**
# This is a capstone project entitled Spytech Web Development Summit.
  # This project is a website for the tech summit and the tech summit aims to establish a strong foundation in tech in my locality.
 - [ ] Watch a video of the project features. https://www.loom.com/share/ca6267236f994290919aa056fa76ebdd
  
- [ ] Bug fix (unexpected generic font-family).
- [ ] New feature ( font-family; Poppins, Arial, Helvetica, sans-serif;)
  - [ ] Run local stylelint config command(npx stylelint "**/*.{css,scss}" --fix  after changing the font.
# Checklist
 [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,72 @@
...
+
+### Usage
+Navigate through the web page to find the information you need. Use the links in the pages to move to different sections.
+
+
+### Deployment
+  Project reside in this Repo. You can clone the repo or download the files manually to your local machine.
+
+### Live Demo
+[Link to my Portfolio](https://spydaspider.github.io/microverse-projects/)
```
Comment (youmari) [inline README.md]: - [ ] You should deploy your capstone project correctly and provide the link here instead of your portfolio link as in the requirements

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @spydaspider , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

✔️ Great Design 
✔️ Professional PR

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by spydaspider** [comment]
Thank you

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @spydaspider, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [x] I suggest that you change the icons colors from black to white, this would help with accessibility in this section:

image

- [x] I suggest that you split the content of each card evenly so all cards match each other, this would help with the design and it will look much better in opinion:

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by spydaspider** [comment]
Thank you, sir. I will attend to the optional changes too at once as it turns out to be true.

**Pull request merged**